Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1919,6 +1951,35 @@ - (void)handleInitSuccess {
     }
 }
 
+- (void) sendOpenNotificationWithOriginalURL:(NSURL*)originalURL
+                              linkParameters:(NSDictionary*)linkParameters
+                                       error:(NSError*)error {
+
+    NSMethodSignature *signature = nil;
```

Comment:
I'm curious why the NSInvocation here. This would be a lot simpler with
```Obj-C
if ([self.delegate respondsToSelector:selector]) {
    [self.delegate branch:self didOpenURL:originalURL linkParameters:linkParameters: error:error];
}
```

I'm sure there's a good reason.